cmd/system-probe/probe.go

```diff
@@ -42,28 +42,28 @@ func CreateSystemProbe(cfg *config.AgentConfig) (*SystemProbe, error) {
 		return nil, fmt.Errorf("%s: %s", ErrSysprobeUnsupported, msg)
 	}
 
-	// make sure debugfs is mounted
-	if mounted, msg := util.IsDebugfsMounted(); !mounted {
-		return nil, fmt.Errorf("%s: %s", ErrSysprobeUnsupported, msg)
-	}
-
 	log.Infof("Creating tracer for: %s", filepath.Base(os.Args[0]))
 
 	t, err := ebpf.NewTracer(config.SysProbeConfigFromConfig(cfg))
 	if err != nil {
 		return nil, err
 	}
 
-	// Setting up the unix socket
-	uds, err := net.NewUDSListener(cfg)
+	var conn net.Conn
+	if runtime.GOOS != "windows" {
+		conn, err = net.NewUDSListener(cfg)
+	} else {
+		conn, err = net.NewListener(cfg)
+	}
+
```

Comment:
Not a fan of this - we can use build tags for this and have a simple `net.NewListener` that uses `NewUDSListener` for linux and something else for windows.